src/gui/widgets/FloatModelEditorBase.cpp

```diff
@@ -385,18 +385,16 @@ void FloatModelEditorBase::enterValue()
 	bool ok;
 	float new_val;
 
-	if (isVolumeKnob() &&
+	if (volKnobType() != VolKnobType::Normal &&
 		ConfigManager::inst()->value("app", "displaydbfs").toInt())
 	{
 		auto const initalValue = model()->getRoundedValue() / 100.0;
 		auto const initialDbValue = initalValue > 0. ? ampToDbfs(initalValue) : -96;
 
-		new_val = QInputDialog::getDouble(
-			this, tr("Set value"),
-			tr("Please enter a new value between "
-					"-96.0 dBFS and 6.0 dBFS:"),
-				initialDbValue, -96.0, 6.0, model()->getDigitCount(), &ok);
-
+		new_val = QInputDialog::getDouble(this, tr("Set value"),
+			tr(volKnobType() == VolKnobType::AbsVol ? "Please enter a new value between -96.0 dBFS and 6.0 dBFS:"
+													: "Please enter a new value between -96.0 dB and 6.0 dB:"),
```

Comment:
The argument to `tr` can only be a string literal, or `lupdate` won't be able to extract the strings for translation. The ternary should be moved outside the `tr` call, with `tr` called separately for each literal.